examples/amm/src/lib.rs

```diff
@@ -42,10 +42,10 @@ pub enum Operation {
     /// add liquidity to the AMM such that you'll be adding AT MOST
     /// `max_token0_amount` of token0, and `max_token1_amount` of token1,
     /// which will be calculated based on the current AMM ratio
-    /// MultiAddress here is the user adding liquidity, which currently can only
+    /// Address here is the user adding liquidity, which currently can only
```

Comment:
```suggestion
    /// `owner` here is the user adding liquidity, which currently can only
```

Reply:
I am not sure which _way_ this should be update - i.e. the comment or the variable name.